modules/webhook-configuring-acs.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-generic-webhooks.adoc
+:_module-type: PROCEDURE
+[id="webhook-configuring-acs_{context}"]
+= Configuring {product-title}
+
+[role="_abstract"]
+Create a new integration in {product-title} by using the webhook URL.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Notifier Integrations* section and select *PagerDuty*.
+. Click *New Integration* (add icon).
+. Enter a name for *Integration Name*.
+. Enter the webhook URL in the *Endpoint* field.
+. If your webhook receiver is using an untrusted certificate, enter a CA certificate in the *CA Cert* box, otherwise leave it blank.
++
+[NOTE]
+====
+The server certificate used by the webhook receiver must be valid for the endpoint DNS name.
+You can turn on the *Skip TLS Verify* toggle to ignore this validation.
+Red Hat does not recommend turning off TLS verification.
+====
+. Optionally, turn on the *Enable Audit Logging* toggle, to receive alerts about all the changes made in {product-title}.
+//See link:/docs/integrate-with-other-tools/enable-audit-logging/[Audit Logging] for more information.
++
+[NOTE]
+====
+Red Hat recommends using separate webhooks for alerts and audit logs to handle these messages differently.
+====
+. To authenticate with the webhook receiver, enter details for:
+** *Username* and *Password* for basic HTTP authentication, or
+** custom *Header* (for example: `Authorization: Bearer <access-token>`).
+. Use *Extra fields* to include additional key-value pairs in the JSON object which {product-title} sends.
```

Comment:
s/which/that
**that** _conjunction_
Use "that", without a comma, to introduce a restrictive clause. Use "which", preceded by a comma, to introduce a
nonrestrictive clause.